staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
Per the commit message for fa8b8b9b5f557d63ecaec8911e680548bf3087a8:
> Exporting `AddToScheme` as a variable holding a method expression is succinct, but leaves the package vulnerable to callers changing its value to some other function. Instead, fix the definition as a function declaration.

Is this a binding that you're expecting callers of the package should be allowed to change?